extension.js

```diff
@@ -99,13 +102,13 @@ function createOverlay() {
   });
   overlayContainer.add_constraint(new Layout.MonitorConstraint({primary: true, work_area: true}));
   Main.layoutManager.addChrome(overlayContainer, {affectsInputRegion: false});
-
-  overlay = new St.Bin({ style_class: '',
-    reactive: true,
-    can_focus: true,
-    x_fill: true,
-    y_fill: false,
-    track_hover: true });
+  
+  // check gnome version to determine correct call to create new overlay 
+  if (gnome_at_least_3_38) {
+    overlay = new St.Bin({ style_class: '', reactive: true, can_focus: true, x_expand: true, y_expand: false, track_hover: true });
```

Comment:
Is there example/docs of x_expand? i didn't find it.

Reply:
I couldn't find docs, but came across this solution with a similar issue experienced by another extension: https://github.com/biji/simplenetspeed/issues/19